### PR TITLE
added links to nav menu

### PR DESCRIPTION
**Pull request by kimberlythegeek**
Addresses Issue #79 

I went ahead and added this to the current branch `gh-pages`, but I don't plan on making too many more changes to the base branch. 

The refactor is complete, and created a new branch from that, `ux-design`, to make it pretty 😉 

I know you guys are busy this week, just a heads up ☺️

### CODE REVIEW

**Comment by m8ttyB** [comment]
r+ lg[reat]tm 💯 
magic

File: index.html [hunk 1]
```diff
@@ -58,6 +58,8 @@
         </ul>
         <ul class="nav nav-sidebar">
           <li><a href="https://quality.mozilla.org/teams/test-engineering/">About Firefox Test Engineering</a></li>
+          <li><a href="http://firefox-test-engineering.readthedocs.io/">ReadTheDocs</a></li>
```
Comment (davehunt) [inline index.html]: s/ReadTheDocs/Documentation/

**Review by davehunt — DISMISSED** [review]

**Comment by davehunt** [comment]
@m8ttyB You _should_ have permissions for submitting review and merging. Please let me know if this is not the case.

**Comment by m8ttyB** [comment]
Spot on @davehunt with href text. r+ for fixing this. It doesn't appear that github provides a mechanism for a team member to mark another reviewers change request as approved. I'm going to dismiss the previous review and r+ this.

@kimberlythegeek can you squash your commits, otherwise I'd be happy to do it for you this time.

**Review by m8ttyB — APPROVED** [review]
r+

**Comment by m8ttyB** [comment]
Merging.

**Pull request merged**